images/linux/scripts/installers/1804/hosted-tool-cache.sh

```diff
@@ -31,14 +31,23 @@ DocumentInstalledItem "Python:"
 pythons=$(ls $AGENT_TOOLSDIRECTORY/Python)
 for python in $pythons; do
 	DocumentInstalledItemIndent "Python $python"
+	# Add symlinks for python, python2 and python3 so that Python can be used out of the box without setup-python
+	if [ ${python:0:3} == "2.7" ] ; then
+		# add python 2.7 for python and python2
+		ln -s $AGENT_TOOLSDIRECTORY/Python/$python/x64/bin/python /usr/local/bin/python
+		ln -s $AGENT_TOOLSDIRECTORY/Python/$python/x64/bin/python /usr/local/bin/python2
+	elif [ ${python:0:3} == "3.6" ] ; then
+		# add python 3.6 for python3 which is the default for 1804
+		ln -s $AGENT_TOOLSDIRECTORY/Python/$python/x64/bin/python /usr/local/bin/python3
+	fi
 done;
 
 # PyPy is also configured using the setup-python action
 pypys=$(ls $AGENT_TOOLSDIRECTORY/PyPy)
 for pypy in $pypys; do
 	DocumentInstalledItemIndent "PyPy $pypy"
 	# Add symlinks for pypy2 and pypy3 to usr/local/bin, there should only be 2 versions of PyPy in the tools cache that is downloaded
-	if [ ${pypy:0:1} -eq "3" ] ; then
+	if [ ${pypy:0:1} == "3" ] ; then
```

Comment:
Updated just for consistency with what we do for Python. Should be the same behavior. In bash if there is a string like "3.5" being compared using `-eq` then it will complain with something like  "Integer expected".